modules/hilary/templates/upstart_hilary.conf.erb

```diff
@@ -17,9 +17,9 @@ limit nofile 32768 32768
 expect fork
 
 # Pass along some environment variables
-env NODE_ENV=production
+env NODE_ENV=<%= @environment %>
```

Comment:
not certain but this seems a duplication, in that we have it set in the init script _and_ in $environment.js